keyboards/gopolar/gg86/info.json

```diff
@@ -0,0 +1,109 @@
+{
+    "keyboard_name": "GG86", 
+    "url": "", 
+    "maintainer": "Gopolar", 
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This name must match the layout macro name (`LAYOUT_tkl_multi_7u` in your code, but you may need to change it to `LAYOUT_all`). For this kind of keyboard you probably should provide multiple layout macros (e.g., something like `LAYOUT_tkl_ansi_tsangan` and `LAYOUT_tkl_iso_tsangan` could be useful).

And it looks like you converted a KLE layout with options, like you have in the pseudographic comment before the layout macro. This is incorrect — the order and number of items in the JSON layout description must be exactly the same as the order and number of arguments in the layout macro. You need to make a KLE layout with ANSI enter and split left and right Shift (as you have in `LAYOUT_tkl_multi_7u`), move the top left key to (0,0) and convert it again to get a proper JSON layout.

Reply:
Could u take a look of the new change I make? I think that should resolved the issue u explained